pkg/template/compiled_template_error.go

```diff
@@ -187,7 +187,7 @@ func (CompiledTemplateMultiError) findClosestLine(ct *CompiledTemplate, posLine
 	}
 }
 
-func (CompiledTemplateMultiError) hintMsg(errMsg string) string {
+func (CompiledTemplateMultiError) hintMsg(errMsg string, err CompiledTemplateError) string {
 	hintMsg := ""
 	switch errMsg {
```

Comment:
👍🏻  Yup, we absolutely need to bring in additional context to ensure we're not giving hints in the wrong circumstances.

💡 At this point, looks like we can "just" take the error, itself, and read the message off of that:

```suggestion
func (CompiledTemplateMultiError) hintMsg(err CompiledTemplateError) string {
	hintMsg := ""
	switch err.Msg {
```

Reply:
Thanks for pointing out this change. I have updated the code.